src/lib/ngsi/ParseData.h

```diff
@@ -237,15 +246,16 @@ typedef struct UpdateContextAvailabilitySubscriptionData
 *
 * UpdateContextData -
 */
-typedef struct UpdateContextData
+struct UpdateContextData
 {
+  UpdateContextData(): ceP(0), entityIdP(0), attributeP(0), contextMetadataP(0), domainMetadataP(0) {}
   UpdateContextRequest   res;
   ContextElement*        ceP;
   EntityId*              entityIdP;
   ContextAttribute*      attributeP;
   Metadata*              contextMetadataP;
   Metadata*              domainMetadataP;
-} UpdateContextData;
+};
```

Comment:
Interesting with all these removal of typedefs here ...
These types are used later in the "mega struct" of ParseData.

I don't know if the compilers have changed, but in the old times, without using 'typedef', e.g.

    typedef struct XXX
    {
        ...
    } XXX;

you couldn't use the typename (without the "struct" keyword).
Has this changed in newer versions of gcc/"C compilers in general"?